flytekit/types/schema/types.py

```diff
@@ -180,6 +180,7 @@ class FlyteSchema(object):
     """
     This is the main schema class that users should use.
     """
+    logger.warning("FlyteSchema is deprecated in 1.4.0, use Structured Dataset instead.")
```

Comment:
Should we mention that the code for it will be removed in the 1.15.0 release?

Reply:
can we hold off?  i don't think we should commit to a release number until we know for sure.  might confuse people

Reply:
so just leave the warning saying that it's deprecated, right?